homeassistant/components/mqtt/__init__.py

```diff
@@ -1206,20 +1206,31 @@ async def websocket_subscribe(hass, connection, msg):
 
     async def forward_messages(mqttmsg: ReceiveMessage):
         """Forward events to websocket."""
+        try:
+            payload = (
+                mqttmsg.payload.decode(DEFAULT_ENCODING)
+                if isinstance(mqttmsg.payload, bytes)
```

Comment:
If it's not `bytes`, what is it?

Reply:
`str` or `None`. Only `bytes` can be decoded.